README.md

```diff
@@ -24,47 +24,105 @@ TODO
  - `npm install --save k-simple-state`
 
 ## API
+### createStore(<store_description>, \<options>)
+| parameter | required | description |
+|---|---|---|
+| <store_description>| required | object describing your store, it can be nested |
+| [\<options>](#options) | optional | all options you may want to override |
+
+#### options
+All options are optionals to keep the default usage as simple as possible.
+
+| key | type | default | description |
+|---|---|---|---|
+| listeners | `array` | `undefined` | array of all listeners. Listeners are a big part of this lib, you can [click here for detail](#TODO). |
+| devtools | `boolean` | `true` | [redux-devtools](https://github.com/zalmoxisus/redux-devtools-extension) is activated. |
+| hideRedux | `boolean` | `true` | actions and selectors from [`k-redux-factory`](https://github.com/alakarteio/k-redux-factory) are used without specifying `dispatch` or `getState`. |
+| enhancer | redux enhancer | `undefined` | usual `compose` and `applyMiddlare` you already use with Redux can be injected here (like router, redux-saga, etc). <br />`compose` and `applyMiddleware` from Redux are exposed by the lib. To use them:<br /> ```import { compose, applyMiddleware } from 'k-simple-state'```. |
+| init | `object` | `undefined` | the default value of your store. |
+
+#### listeners
 TODO
 
 ## Examples
-Full example
-```es6
-const { createState } = require('k-simple-state')
+### Simple one, with k-redux-factory
```

Comment:
You should not speak about `k-redux-factory` here, but considering that `keyValue`, `simpleObject` are default behaviour are coming from `k-simple-state`.

*I explain why:*
I'm a user discovering this lib (k-simple-state) and in the first example I see that it's depending on an other one that I have to watch: 2 things to learn _(I'm lazy)_

IMO if you present those functions / concepts as a default behaviour of `k-simple-state`, and make references to `k-redux-factory` deeper in the documentation (or hide it completely), it will be less complicated to start with.

Reply:
Ok, I can do that in the first layer of documentation.
But I have to speak about k-redux-factory at some point to avoid documentation duplication. (Or maybe we can simply link to TYPES.md from k-redux-factory)